<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>687</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PALADIN</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 8pt &quot;Garamond&quot;;
background-color: rgb(30, 58, 138);
background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createAccountButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>460</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
background-color: rgb(44, 44, 44);
font-size:33pt;
font-weight: 700;

font: 75 12pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../photos/paladin.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>390</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;

font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="repeatPasswordTextbox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>390</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(237, 235, 225);
font: 75 16pt &quot;Times New Roman&quot;;
background-color: rgb(44, 62, 80);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 215, 0);
font-size:33pt;
font-weight: 700;
background-color: rgb(44, 44, 44);
font: 75 16pt &quot;Times New Roman&quot;;
</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
